src/App.tsx

```diff
@@ -11,11 +11,15 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState(state => ({
```

Comment:
use decomposing structure on next time)
```suggestion
    this.setState(({ count })=> ({
```